### ISSUE TITLE
Include wikitext from /all response in /article response

### ISSUE THREAD

**Issue by harej** [internetarchive/iari#808]
I currently use the `/all` endpoint for the wikitext of the references. I would use the `/article` endpoint, but it does not contain the wikitext content I need.

**Comment by dpriskorn** [comment]
You can get the wikitext of each reference from the references and reference endpoint.
The all endpoint is not reliable nor tested nor efficient so it should not be used.

**Comment by harej** [comment]
I was asking you to add the content to `/article` SO THAT I CAN STOP USING `/all`.

**Comment by mojomonger** [comment]
I have another reason for adding the WikiText to the /article endpoint:

Currently, to display the text for the references in the reference list display, i am pulling data from the reference data in the /dehydrated_references . 

When there is a "named" reference, the only place i can get that "name" information is from inside the wikitext, which is only available AFTER i fetch the full reference details. I want to display the name before.

For this use, either i will either need the wikitext, or, if the reference is a named reference, provide the name in a "name" field. or something. we need to finalize the format.

i know the property field is called "dehydrated_references", but maybe we should reconsider that for the default reference data for /article

**Comment by mojomonger** [comment]
Just for more information regarding reference list in /article endpoint:
For the uses in IARE, all that is needed is a "name" field in the dehydrated_reference object, filled with the reference name if there is one. This will facilitate "linking" to the native reference a named reference refers to.

**Comment by dpriskorn** [comment]
Closing this again. 
As I just wrote in slack the purpose of the article endpoint is to give a light overview of the article
The references endpoint is used to get some or all of the references for a give article in ONE request.

The all endpoint is not needed nor desired.

Putting all the references information into the article endpoint is not desirable because it breaks a basic design constraint of max 1mb json response.
